tasks/publish_statistics.rb

```diff
@@ -50,15 +50,9 @@
 
     logger.info("Creating #{adverbial} metrics for S3 with #{args[:date]}")
 
-    metrics = Metrics::ActiveUsers.new(period: period, date: args[:date])
-
-    logger.info("[#{metrics.key}] Generating metrics...")
-
-    metrics.generate!
-
-    logger.info("[#{metrics.key}] Uploading metrics...")
-
-    metrics.publish!
+    logger.info("[#{metrics.key}] Fetching and uploading metrics...")
```

Comment:
what's `metrics` here?

Reply:
That was a little bit of a mistake there. Fixed it now...